app/models/spree/order_decorator.rb

```diff
@@ -304,6 +304,14 @@ def tax_adjustment_totals
     end
   end
 
+  def price_adjustments
```

Comment:
What is this method doing in this commit?

Reply:
Ah, that looks like it shouldn't be in that commit. I think it was a method that we depend on in the decorator, but got removed from the model in `Spree 2.1`.

Reply:
I've moved it to a new commit with a message.